src/components/home/Search.tsx

```diff
@@ -0,0 +1,191 @@
+import Button from '../common/Button';
+import SearchBottom from './SearchBottom';
+import SearchCheckedBox from './SearchCheckedBox';
+import SearchClickedFieldWrapper from './SearchClickedFieldWrapper';
+import SearchRadioButton from './SearchRadioButton';
+import SearchTextField from './SearchTextField';
+import { CalendarGreyIcon, OriginIcon, PinIcon, ProfileIcon } from '@/assets/svg';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+const Search = () => {
+	const [selectedRadio, setSelectedRadio] = useState<string>('왕복');
+	const handleRadioClick = (value: string) => {
+		setSelectedRadio(value);
+	};
+
+	const [selectedNearbyDeparture, setSelectedNearbyDeparture] = useState(false);
+	const [selectedNearbyArrival, setSelectedNearbyArrival] = useState(false);
+	const [selectedDirect, setSelectedDirect] = useState(false);
+
+	const [departure, setDeparture] = useState('');
+	const [arrival, setArrival] = useState('');
+
+	const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
p4: 현재 모달이 `open`, `close`되는 것이 아니라, `SearchBottom` 컴포넌트가 `open`, `close`가 되고 있기 떄문에 네이밍이 길어지더라도보다 직관적으로 다음과 같이 수정하는 것이 어떨까요? 
```suggestion
	const [isSearchBottomOpen, setIsSearchBottomOpen] = useState(false);
```